app/models/audit/event/whoami.rb

```diff
@@ -0,0 +1,80 @@
+module Audit
+  module Event
+    class Whoami
+
+      #ACCOUNT_PLACEHOLDER = "ACCOUNT_MISSING"
+      #USERNAME_PLACEHOLDER = "USERNAME_MISSING"
+
+      # Note: "username" may refer to a user or host.
+      def initialize(
+        client_ip:,
+        role_id:,
+        success:
+      )
+        @client_ip = client_ip
+        @role_id = role_id
+        @success = success
+
+        # Implements `==` for audit events
+        @comparable_evt = ComparableEvent.new(self)
+      end
+
+      # Note: We want this class to be responsible for providing `progname`.
+      # At the same time, `progname` is currently always "conjur" and this is
+      # unlikely to change.  Moving `progname` into the constructor now
+      # feels like premature optimization, so we ignore reek here.
+      # :reek:UtilityFunction
+      def progname
+        Event.progname
+      end
+
+      def severity
+        attempted_action.severity
+      end
+
+      def to_s
+        message
+      end
+
+      def message
+        "#{@role_id.role_id} checked identity using WhoAmI"
```

Comment:
Please ask from @shulifink  to review this text

Reply:
Regarding @egvili 's comment:
> 1. The ticket to use for the PR should be Git ticket. You can use #1987

There's a dedicated ticket for this: https://github.com/cyberark/conjur/issues/2052

Reply:
> Please ask from @shulifink to review this text
she already did.

Reply:
i wouldn't write `WhoAmI` but `whoami` as this is the command. Also - `checked its identity` seems better.
In summary, i think we should write `#{@role_id.role_id} checked its identity using the whoami command`

Reply:
This text was already reviewed and changed by Chuli, so, I'm not sure about changing it again.